src/esp/io/JsonEspTypes.cpp

```diff
@@ -49,6 +49,16 @@ JsonGenericValue toJsonValue(const gfx::replay::Keyframe& keyframe,
     io::addMember(obj, "userTransforms", userTransformsArray, allocator);
   }
 
+  if (keyframe.lights && !keyframe.lights->empty()) {
+    JsonGenericValue lightsArray(rapidjson::kArrayType);
+    for (const auto& light : *keyframe.lights) {
+      JsonGenericValue lightObj(rapidjson::kObjectType);
+      io::addMember(lightObj, "light", light, allocator);
+      lightsArray.PushBack(lightObj, allocator);
```

Comment:
```suggestion
      lightsArray.PushBack(std::move(lightObj), allocator);
```
I am not sure if it actually defines a move ctor, but if clang-tidy doesn't complain this should be what you want to do.

Reply:
`clang-tidy` complained, so I removed the `std::move` here.

Reply:
Ah, so it is trivially copyable. NVM